src/codeu/chat/client/ClientUser.java

```diff
@@ -84,7 +84,7 @@ public boolean signInUser(String name, String password) {
 	    return (prev != current);
     }
 
-    System.out.println("Login was UNSUCCESSFUL due to your username and password combinatio of: " + name + " " + password);
+    System.out.println("Login was UNSUCCESSFUL due to your username and password combination of: " + name + " " + password);
```

Comment:
Probably want to use log statements, and you probably don't want to communicate to the user view the console. You probably also don't want to include passwords in the log.